_config.yml

```diff
@@ -20,6 +20,7 @@ header_pages:
   - url: /examples/
     title: Examples
   - showcases.md
+  - integrations.md
```

Comment:
Hmm, this makes the list a bit too long so it wraps before it disappears into the hamburger menu...

I guess the alternatives is:
1. Use a shorter name
2. Use a smaller font
3. Change the breakpoints so it collapse to the hamburger menu earlier
4. Let it wrap

I'm not sure what alternative I think is the best...

Reply:
I take the number 2!

Reply:
Sounds great!